collect_app/src/androidTest/java/org/odk/collect/android/test/FormLoadingUtils.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2019 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.test;
+
+import android.content.Intent;
+import android.content.res.AssetManager;
+import android.os.Environment;
+
+import org.apache.commons.io.IOUtils;
+import org.odk.collect.android.activities.FormEntryActivity;
+import org.odk.collect.android.tasks.FormLoaderTask;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.core.app.ApplicationProvider;
+import androidx.test.espresso.intent.rule.IntentsTestRule;
+
+import static org.odk.collect.android.activities.FormEntryActivity.EXTRA_TESTING_PATH;
+
+public class FormLoadingUtils {
+    private static final String FORMS_PATH = Environment.getExternalStorageDirectory().getPath() + "/odk/forms/";
```

Comment:
Can't you use `FORMS_PATH` from `Collect` class instead of creating this field?

Reply:
Aha, great call!